lib/importer/import-default.js

```diff
@@ -0,0 +1,287 @@
+"use strict";
+
+const stepStates = require('./import-step-states'),
+        execSync = require('child_process').execSync,
+         rfc6902 = require('rfc6902'),
+            path = require('path'),
+              fs = require('fs-extra');
+
+/**
+ * Processes metadata located in <pkg-name@version?.json> metadata file
+ *
+ * Steps in execution order
+ * 0. register
+ *
+ * 1. applyPatches
+ * 2. registerDependencies
+ * 3. registerBundles
+ * 4. saveProject
+ * 5. installTasks
+ * 6. executeScripts
+ */
+module.exports = class {
+
+    constructor() { }
+
+    /**
+     * Register this class with ImportEngine
+     *
+     * @param engine ImportEngine ImportEngine instance
+     * @param options any given CLI arguments
+     * @param projectInfo basic information about the project itself
+     *
+     */
+    register(engine, options, projectInfo) {
+        this.engine = engine;
+        this.cliParams = options;
+        this.projectInfo = projectInfo;
+        this.ui = this.engine.cliBridge.ui;
+
+        this.bundles = this.engine.projectConfig.build.bundles;
+        if (this.bundles.length === 0) {
+            throw new Error("aurelia.json: bundles section is missing.");
+        }
+
+        this.bundleName = this.cliParams.bundle || 'vendor-bundle.js';
+        this.bundle = this.bundles.find(item => item.name === this.bundleName);
+    }
+
+    /**
+     * Apply RFC6902 standard JSON patches against aurelia.json
+     *
+     * @return ImportStepState | void
+     */
+    applyPatches() {
+        let project = this.engine.projectConfig,
+            config = this.engine.definitionConfig;
+
+        // patch aurelia.json settings with overrides
+        if (this.cliParams.action === 'install') {
+            if (config && config.patches && config.patches.length > 0) {
+
+                let result = rfc6902.applyPatch(project, config.patches),
+                    errors = result.filter(err => err !== null);
+
+                if (errors.length > 0) {
+                    let msg = errors.map(e => e.message).join(', ');
+                    throw new Error(`An error occurred while applying custom settings.\nErrors: ${msg}`);
+                }
+
+                this.writeLine(`[INFO] Custom project settings have been applied to aurelia.json.`);
+            }
+        }
+    }
+
+    /**
+     * Register dependencies within aurelia.json bundle section
+     * @return ImportStepState | void
+     */
+    registerDependencies() {
+        if (!this.bundle) {
+            this.writeLine(`[INFO] Bundle '${this.bundleName}' could not be found. Looking for default bundles...`);
+
+            // There are 2 sections by default, second is usually the vendor-bundle.js
+            // Although, some developers prefer to merge everything into a single bundle
+            let index = this.bundles.length > 1 ? 1 : 0;
```

Comment:
See note above. If there's more than one bundle, and one is not specified, the lets use the bundle with the largest number of dependencies.